sentences/et/_common.yaml

```diff
@@ -0,0 +1,13 @@
+language: et
+responses:
+  errors:
+    no_intent: "Vabandust, ma ei saanud aru"
+    no_area: "Puudub ala nimega {{ area }}"
+    no_domain: "{{ area }} ei sisalda {{ domain }}"
+    no_device_class: "{{ area }} ei sisalda {{ device_class }}"
+    no_entity: "Puudub seade või üksus nimega {{ entity }}"
```

Comment:
I think "entity" has been translated to "olem" in other contexts. Weird word, but in my opinion "üksus" isn't much better, I'd recommend keeping with the previous translations for consistency's sake.

Reply:
First time I see word "olem", which contexts you referring? I checked my doubts with Google translator

Reply:
I'm just thinking if a person has their UI set to Estonian, and is used to seeing this word in settings and throughout the menu, they would understand the error better than if a different word is used.

![Screenshot_20230515_082502_Home Assistant](https://github.com/home-assistant/intents/assets/3413870/1a83be45-250a-4b5a-b7df-543cfa2757e7)

Üksus does seem better than either, indeed. But I would suggest keeping with the HA translation for now and perhaps propose the change in a PR separately so it's changed everywhere at once.

Reply:
I must admit, I never switched HA to Estonian language. All those new IT words in Estonian language are hard to digest, so I prefer English in UI. 
Still, I really don't like to speak with my home in foreign language. 
I hope, HA voice assistant finally leads to working vocal communication in our native language and I try to give my part. 
First step would be to see Estonian in Supported Languages list. Hopefully more fellow countryman will join then for fine tuning.

Reply:
It's exactly the same for me. I only switched.to English for checking for consistency.

Anyhow, if you prefer not to change, I support the idea of just getting the language added and building on top of that. 

I did too much in my PR and it got stale 😅

Reply:
I looked your repository and you have done good job. I you don't mind, I try to take parts from there and continue testing.

Reply:
Of course, feel free. I'm currently adjusting to life with 2 kids,. youngest being 2 weeks old, so I've put my PR on hold 😅